dkpro-core-lancaster-asl/src/main/java/org/dkpro/core/lancaster/LancasterStemmer.java

```diff
@@ -55,6 +59,15 @@
     @ConfigurationParameter(name = PARAM_STRIP_PREFIXES, mandatory = false, defaultValue = "false")
```

Comment:
A primitive boolean parameter must have set `mandatory` to `true` as `null` is not a valid value.